test/lib/ls-utils.test.ts

```diff
@@ -212,3 +215,14 @@ test("parse ls output", async (t) => {
     });
   });
 });
+
+const getLSOutputFixture = (file: string) =>
```

Comment:
I think we try to avoid arrow functions, leave it if you prefer it this way but I think it's better to stick with regular functions.

Reply:
That seems to make "prettier" upset :upside_down_face: 

```
ERROR: 219:28  only-arrow-functions  non-arrow functions are forbidden
```